addons/mail/static/tests/tours/mail_full_composer_test_tour.js

```diff
@@ -0,0 +1,73 @@
+odoo.define('mail/static/tests/tours/mail_full_composer_test_tour.js', function (require) {
+"use strict";
+
+const tour = require('web_tour.tour');
+
+/**
+ * This tour depends on data created by python test in charge of launching it.
+ * It is not intended to work when launched from interface. It is needed to test
+ * an action (action manager) which is not possible to test with QUnit.
+ * @see mail/tests/test_mail_full_composer.py
+ */
+tour.register('mail/static/tests/tours/mail_full_composer_test_tour.js', {
+    test: true,
+}, [{
+    content: "Click on Send Message",
+    trigger: '.o_ChatterTopbar_buttonSendMessage',
+},{
+    content: "Write something in composer",
+    trigger: '.o_ComposerTextInput_textarea',
+    run: 'text blahblah',
+},{
+    content: "Open full composer",
+    trigger: '.o_Composer_buttonFullComposer',
+},{
+    content: "Check subject is autofilled",
+    trigger: 'input[name="subject"]',
+    run: () => {
+        const subjectValue = document.querySelector('input[name="subject"]').value;
+        if (subjectValue !== "Re: Test User") {
+            tour._consume_tour(
+                tour.running_tour,
+                `Full composer should have "Re: Test User" in subject input (actual : ${subjectValue})`
+            );
+        }
+    },
+},{
+    content: "Check composer content is kept",
+    trigger: '.oe_form_field[name="body"]',
+    run: () => {
+        const bodyContent = document.querySelector('.oe_form_field[name="body"] textarea').textContent;
+        if(!bodyContent.includes("blahblah")) {
+            tour._consume_tour(
+                tour.running_tour,
+                `Full composer should contain "blahblah" in body input (actual : ${bodyContent})`
+            );
+        }
+    },
+},{
+    content: "Open templates",
+    trigger: '.o_field_widget[name="template_id"] input',
+    run: 'click',
+}/*,{ //FIXME should check that at least one template is listed ...
+    content: "Check a template is listed",
+    trigger: 'ul.ui-widget-content',
+    run: () => {
+        const dropdownElements = document.querySelectorAll('ul.ui-widget-content li');
+        const found = dropdownElements.reduce((acc, elem) => acc || elem.textContent.includes('Test template'), false);
+        if (!found) {
+            tour._consume_tour(
+                tour.running_tour,
+                "Full composer should contain at least one mail template"
+            );
+        }
+    },
```

Comment:
Issue is due to magic in tour manager that makes selector only look inside modal DOM when a modal is opened.
Problem is that autocomplete is outside of modal, so a special flag has to be added.

```suggestion
    content: "Check template is listed",
    trigger: '.ui-autocomplete .ui-menu-item a:contains(Test template)',
    in_modal: false,
```